docs/GettingStarted.md

```diff
@@ -511,6 +512,7 @@ For a list of available integrations, and their configuration options, please re
 | Redis                      | `redis`                    | `>= 3.2`                 | `>= 3.2`                 | *[Link](#redis)*                    | *[Link](https://github.com/redis/redis-rb)*                                    |
 | Resque                     | `resque`                   | `>= 1.0`                 | `>= 1.0`                  | *[Link](#resque)*                   | *[Link](https://github.com/resque/resque)*                                     |
 | Rest Client                | `rest-client`              | `>= 1.8`                 | `>= 1.8`                  | *[Link](#rest-client)*              | *[Link](https://github.com/rest-client/rest-client)*                           |
+| Roda                       | `roda`                     | `>= 2.1`                 | `>= 2.1`                  | *[Link](#roda)*                     | *[Link](https://github.com/jeremyevans/roda)*                                  |
```

Comment:
If there's an upper limit on versions we support, we should add that here as well.

Reply:
Great catch! I was being cautious by hardcoding a limit of 4.x, so I should update this Getting Started doc too!

Reply:
In the latest commit, this now gives a range ending <4 (since it works with 3.x):
```
| Roda                       | `roda`                     | `>= 2.1, <4`             | `>= 2.1, <4`              | *[Link](#roda)*                     | *[Link](https://github.com/jeremyevans/roda)*   
```